src/theme/Tabs/index.tsx

```diff
@@ -28,7 +28,7 @@ export default function TabsWrapper(props: Props): ReactNode {
         ...child,
         props: {
           ...child.props,
-          value: nanoid(),
+          value: useId(),
```

Comment:
The number of calls to hooks must be the same between each re-render, so this will break if `props.children` ever has a different number of children between renders.

A similar problem is described in [the docs for `useRef`](https://react.dev/learn/manipulating-the-dom-with-refs#how-to-manage-a-list-of-refs-using-a-ref-callback). The solution they use there won't work here, as it's dependent on each item having a unique ID, which is the thing we want to generate for each item.

I don't know how exactly `<TabItem>` and `<Tabs>` work, but I wonder if we could get away with using an array index as the value. Alternatively, if `value` needs to be unique within some broader context, call `useId` once within the render body of `TabsWrapper` and then append an index to it here.